octopus.go

```diff
@@ -18,9 +18,15 @@ not, you can obtain one from Tidepool Project at tidepool.org.
 package main
 
 import (
+	"crypto/tls"
```

Comment:
Good cleanup.  We should standardize on putting standard libraries first in import.  Though, does go fmt mess with this?